src/extrovermectin.ts

```diff
@@ -208,19 +220,46 @@ function initializeDireWarren(): void {
   if (options.some((option) => banishedMonsters.get(option) === $monster`fluffy bunny`)) {
     return;
   }
-  if (banishedMonsters.get($skill`Furious Wallop`) === $monster`fluffy bunny`) return;
+  if (banishedMonsters.get($skill`Batter Up!`) === $monster`fluffy bunny`) return;
 
   if (!have($item`miniature crystal ball`)) {
     options.push(...$items`Louder Than Bomb, tennis ball`);
   }
-  const banish = options.sort((a, b) => mallPrice(a) - mallPrice(b))[0];
-  acquire(1, banish, 50000, true);
-  do {
-    adventureMacro(
-      $location`The Dire Warren`,
-      Macro.if_($monster`fluffy bunny`, Macro.item(banish)).step(embezzlerMacro())
-    );
-  } while ("fluffy bunny" !== get("lastEncounter"));
+  const canBat = myFury() >= 5 && have($skill`Batter Up!`);
+  if (canBat) {
+    if (have($skill`Iron Palm Technique`) && !have($effect`Iron Palms`)) {
```

Comment:
this is inverted logic. iron palms screws up clubs, you explicitly do not want iron palms on if you want to cast batter up.